packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -1042,6 +1089,76 @@ export default class TabAdvanced extends Component {
                     </td>
                   </tr>
                 )}
+                {vm.boot.firmware === 'uefi' && (
+                  <tr>
+                    <th>{_('secureBoot')}</th>
+                    <td>
+                      <Toggle value={vm.secureBoot} onChange={value => editVm(vm, { secureBoot: value })} />
+                      <a
+                        className='text-muted'
+                        href='https://xcp-ng.org/docs/guides.html#guest-uefi-secure-boot'
+                        rel='noreferrer'
+                        style={{ display: 'block' }}
+                        target='_blank'
+                      >
+                        <Icon icon='info' /> {_('secureBootLinkToDocumentationMessage')}
+                      </a>
+                    </td>
+                  </tr>
+                )}
+                {vm.boot.firmware === 'uefi' &&
+                  semver.satisfies(host?.version, '>=8.3.0') && [
+                    <tr key='secureBootStatus'>
+                      <th>{_('secureBootStatus')}</th>
```

Comment:
As we raised the update delay for the values of this field to several tens of seconds, we should make it so it is force-refreshed whenever someone changes the value of the "Secure Boot" on/off button, otherwise it looks like it wasn't applied.